pkg/scheduler/framework/plugins/defaultbinder/default_binder_test.go

```diff
@@ -66,7 +67,8 @@ func TestDefaultBinder(t *testing.T) {
 				return true, gotBinding, nil
 			})
 
-			fh, err := frameworkruntime.NewFramework(nil, nil, nil, frameworkruntime.WithClientSet(client))
+			var pr config.KubeSchedulerProfile
+			fh, err := frameworkruntime.NewFramework(nil, pr, frameworkruntime.WithClientSet(client))
```

Comment:
nit:
```suggestion
			fh, err := frameworkruntime.NewFramework(nil, config.KubeSchedulerProfile{}, frameworkruntime.WithClientSet(client))
```

Unless we make `NewFramework` accept a pointer and make it tolerant to nil?